process/process_linux.go

```diff
@@ -1236,7 +1235,7 @@ func (p *Process) fillFromTIDStatWithContext(ctx context.Context, tid int32) (ui
 		System: float64(stime / ClockTicks),
 	}
 
-	bootTime, _ := host.BootTime()
+	bootTime, _ := common.BootTime()
```

Comment:
Remove the `BootTime()` definition in the `common` package and instead propagate the passed `ctx`variable with `common.BootTimeWithContext(ctx)` here.